source/magpie/dataframe.d

```diff
@@ -1023,6 +1024,9 @@ public:
     auto opIndex(Args...)(Args args)
         if(Args.length > 0 && Args.length < 3)
     {
+        static assert(is(Args[0] == string[]), "Indexes must be an array of string");
+        static if(Args.length > 1)
+            static assert(is(Args[1] == int), "Mention axis as 0 if you want to fetch a row");
```

Comment:
That message could use some rewording.

Reply:
Addressed in cbe18d9
Improved the message explaining the correct usage for binary operations.